spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -124,10 +128,20 @@
     normalized_params[:bcc].should eq []
   end
 
+  it 'distinguishes between angle brackets in the display name and the email address' do
+    params = default_params.merge(
+      to: '"<Display name>" <email@example.com>',
+      envelope: '{"to":["email@example.com"]}',
+    )
+    normalized_params = normalize_params(params)
+
+    normalized_params[:bcc].should eq []
```

Comment:
ugh I guess I didn't realize there's some weird bcc thing going on because... sendgrid doesn't send bcc?

Anyway I like your suggested spec here, I'd prefer it to the weird bcc one.

It occurs to me that leaving the angle brackets in names in there might cause issues for `griddler` itself, considering [it "parses" email addresses by splitting at angle brackets](https://github.com/thoughtbot/griddler/blob/master/lib/griddler/email_parser.rb#L62):

```ruby
  def self.extract_email_address(full_address)
    full_address.split('<').last.delete('>').strip
  end
```

Sigh... why don't we make griddler depend on the [mail](https://github.com/mikel/mail/) gem and use [`Mail::Address.new` to parse email addresses](http://www.rubydoc.info/github/mikel/mail/Mail/Address#initialize-instance_method)?